### PR TITLE
SP-412-healthchecks

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

File: common/config.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      timeout: {
+        response: get('AUTH_ENDPOINT_TIMEOUT_RESPONSE', 10000, true),
+        deadline: get('AUTH_ENDPOINT_TIMEOUT_DEADLINE', 10000, true),
+      },
+      agent: {
+        maxSockets: 100,
+        maxFreeSockets: 10,
+        freeSocketTimeout: 30000,
+      },
+      apiClientId: get('API_CLIENT_ID', 'use-of-force-client', true),
```
Comment (andrewrlee) [inline common/config.js]: do you have a sentence planner client id?

File: common/services/healthcheck.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const buildInfo = {
+    uptime: process.uptime(),
+    build: buildInformation,
+    version: buildInformation && buildInformation.buildNumber,
+  }
+
+  return { ...result, ...buildInfo }
+}
+
+function getBuild() {
```
Comment (andrewrlee) [inline common/services/healthcheck.js]: meh.. is this never not present?  if it is could potentially put in a default file which is overwritten by the build?

Comment (jonhindle) [reply common/services/healthcheck.js]: I agree about putting in a local default. I appreciate its an unlikely absentee, I'm inclined to just leave it though; added robustness against the improbable.

File: common/config.js (hunk 1 above)
Comment (jonhindle) [reply common/config.js]: Done.

**Pull request merged**